test/integration/simple_operations_test.lua

```diff
@@ -1337,3 +1337,89 @@ for op, case in pairs(gh_328_error_cases) do
         end
     end
 end
+
+pgroup.test_noreturn_opt = function(g)
+    -- correct insert with noreturn
+    local result, err = g.cluster.main_server.net_box:call( 'crud.insert', {
+        'customers', {1, box.NULL, 'Elizabeth', 24}, {noreturn = true}
+    })
+    t.assert_equals(err, nil)
+    t.assert_equals(result.rows, {})
```

Comment:
The API should be the same everywhere -- result here should be nil as well

Reply:
Done